src/components/mask-country-manager/mask-country-manager.js

```diff
@@ -67,10 +68,10 @@ const MaskCountryManager = props => {
   }, [countryExtent, isCountryMode]);
 
   useEffect(() => {
-    if (graphicsLayer && !countryISO) {
+    if (graphicsLayer && (!countryISO || sceneMode === GLOBAL_SCENE)) {
```

Comment:
You can use `isCountryMode` prop, it's already there

Reply:
🤦